core/src/wallet/ripple/synchronizers/RippleLikeAccountSynchronizer.cpp

```diff
@@ -0,0 +1,568 @@
+/*
+ *
```

Comment:
I think as we don't need a lot of methods in AbstractBlockchainExplorerAccountSynchronizer.h any more.
I don't know whether we can do a cleanup for AbstractBlockchainExplorerAccountSynchronizer.h

Reply:
If the idea is to create independant synchronizer for each coin, we can just delete the file once it is done